filament/backend/include/backend/DriverEnums.h

```diff
@@ -281,7 +281,8 @@ enum class PixelDataType : uint8_t {
     INT,
     HALF,
     FLOAT,
-    COMPRESSED
+    COMPRESSED,
+    UINT_10F_11F_11F_REV,
```

Comment:
Why rev? Is that the internal GL format? So it's BGR right?

Reply:
yes